docs/source/_includes/runner_parameters/winrm_cmd.rst

```diff
@@ -0,0 +1,14 @@
+.. NOTE: This file has been generated automatically, don't manually edit it
```

Comment:
Did you manually create this file? It _should_ get generated by https://github.com/StackStorm/st2docs/blob/master/scripts/generate-runner-parameters-documentation.py, called by the Makefile here: https://github.com/StackStorm/st2docs/blob/master/Makefile#L67

Obviously this bit won't work until the runners actually get checked in though

Reply:
Yes, i manually cloned down my `st2` fork and checked out my `feature/winrm-runner` branch.

Should i delete this file?

Reply:
Doing a manual clone/branch was fine for testing, but they should not be included in the PR. They _should_ get auto-generated once the upstream changes are in place. For now, that will result in build errors here.

I think it will be 2.9 that we merge those, so we won't merge this docs change until after 2.8 release (which is not far away)